03.asynchronous/promise.js

```diff
@@ -0,0 +1,67 @@
+import timers from "timers/promises";
+import sqlite3 from "sqlite3";
+const db = new sqlite3.Database(":memory:");
+
+//Promise用のメソッド作成
```

Comment:
`//` のあとにはスペースを入れてください。

```suggestion
// Promise用のメソッド作成
```

Reply:
[📃 chore: コメント削除](https://github.com/reckyy/js-practices/pull/3/commits/faab14f19d191e668c4094e8876dcf6e4c5a4f16)